src/main/java/uk/gov/ons/fwmt/fwmtrmadapter/service/impl/MessageConverterImpl.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.ons.fwmt.fwmtrmadapter.service.impl;
+
+import org.springframework.stereotype.Component;
+import uk.gov.ons.ctp.response.action.message.instruction.ActionAddress;
+import uk.gov.ons.ctp.response.action.message.instruction.ActionInstruction;
+import uk.gov.ons.ctp.response.action.message.instruction.ActionRequest;
+import uk.gov.ons.fwmt.fwmtgatewaycommon.Address;
+import uk.gov.ons.fwmt.fwmtgatewaycommon.FWMTCancelJobRequest;
+import uk.gov.ons.fwmt.fwmtgatewaycommon.FWMTCreateJobRequest;
+import uk.gov.ons.fwmt.fwmtrmadapter.service.MessageConverter;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+@Component
+public class MessageConverterImpl implements MessageConverter {
+
+  @Override public FWMTCreateJobRequest createJob(ActionInstruction actionInstruction) {
+    FWMTCreateJobRequest fwmtCreateJobRequest = new FWMTCreateJobRequest();
+    ActionRequest actionRequest = actionInstruction.getActionRequest();
+    ActionAddress actionAddress = actionRequest.getAddress();
```

Comment:
Is address a mandatory value in ActionInstruction message? If yes, check for NPE and throw error accordingly